snpeff/5.1/Dockerfile

```diff
@@ -0,0 +1,70 @@
+ARG SNPEFF_VER="5.1d"
+ARG SNPSIFT_VER="5.1"
+
+FROM ubuntu:focal as app
+
+ARG SNPEFF_VER
+ARG SNPSIFT_VER
+
+# Metadata
+LABEL base.image="ubuntu:focal"
+LABEL dockerfile.version="1"
+LABEL software="SnpEff & SnpSift"
+LABEL snpeff.software.version=$SNPEFF_VER
+LABEL snpsift.software.version=$SNPSIFT_VER
+LABEL description.SnpEff="Genetic variant annotation and effect prediction toolbox."
+LABEL description.SnpSift="Used after SnpEff annotation to filter and manipulate annotated files."
+LABEL website="https://pcingola.github.io/SnpEff"
+LABEL license="https://github.com/pcingola/SnpEff/blob/master/LICENSE.md"
+LABEL maintainer="Tom Iverson"
+LABEL maintainer.email="tiverson@utah.gov"
+
+# Install open using apt 
+RUN apt-get update && apt-get install -y software-properties-common &&\
+  apt-get update && apt-get install -y \
+  openjdk-11-jre \
+  ant \
+  maven \
+  wget \
+  unzip
```

Comment:
Could you add 
```
&& \
 apt-get autoclean && rm -rf /var/lib/apt/lists/*
 ```
 
 to the end of the apt-get installation step? This helps reduce the size of the container.